code/modules/projectiles/guns/rifles.dm

```diff
@@ -928,7 +928,7 @@
 	gun_firemode_list = list(GUN_FIREMODE_SEMIAUTO, GUN_FIREMODE_BURSTFIRE, GUN_FIREMODE_AUTOBURST)
 	attachable_offset = list("muzzle_x" = 32, "muzzle_y" = 18,"rail_x" = 12, "rail_y" = 23, "under_x" = 23, "under_y" = 15, "stock_x" = 22, "stock_y" = 12)
 
-	fire_delay = 0.5 SECONDS
+	fire_delay = 0.3 SECONDS
```

Comment:
the weapon uses burst fire so you might want to change the burst delay especifically(there is delay between the bullets in a burst and delay between bursts btw)